pkg/scheduler/factory/factory_test.go

```diff
@@ -68,7 +72,11 @@ func TestCreateFromConfig(t *testing.T) {
 	client := fake.NewSimpleClientset()
 	stopCh := make(chan struct{})
 	defer close(stopCh)
-	factory := newConfigFactory(client, v1.DefaultHardPodAffinitySymmetricWeight, stopCh)
+	framework, err := framework.NewFramework(framework.NewRegistry(), nil, []config.PluginConfig{})
+	if err != nil {
+		t.Errorf("error initializing the scheduling framework: %v", err)
```

Comment:
If you can't initialize the framework it seems like you'd want a `t.Fatalf` instead of just a `t.Errorf`. Is it the norm that the test will continue and provide meaningful feedback after a failure here?

Reply:
Same comment applies to other similar locations

Reply:
I agree, it makes more sense to use a `t.Fatalf` instead.